sips/cluster_consensus.md

```diff
@@ -28,143 +41,156 @@ The only validator-dependent field is `CommitteeIndex` and it does not have to b
 
 For the `Sync Committee` duty, operators agree on a `phase0.Root` data which is also independent of the validator.
 
-On the other hand, a post-consensus phase is still required for each duty that was previously aggregated. Thus, the partial signatures for each validator must still be shared between parties. Better than sending a message for each eth duty, we recommend merging the post-consensus partial signatures into a single message.
 
-## Improvement
+## Spec changes
 
-This proposal helps to decrease the number of messages exchanged in the network and the processing cost.
+### Design
 
-According to Monte Carlo simulations using a dataset based on the Mainnet, this proposal reduces to $21.60$% the current number of messages exchanged in the network. Note that this result includes aggregating the post-consensus messages into a single message.
+Under the new design we will have a `Cluster` object that will be the top level object in charge of processing consensus messages and partial signature messages for the attestation and sync committee roles.
 
-Regarding the number of bits exchanged, we estimate that this proposal will reduce the current value to, at least, $52.96$%. Notice that this reduction is not as significant as the number of messages reduction due to the larger post-consensus messages.
+The `Cluster` will hold a single `ConsensusRunner` and multiple `PartialSigRunners` for each Validator managed by the cluster.
 
-Again with Monte Carlo simulations using the Mainnet dataset, the number of attestation duties aggregated presented the following distribution (notice that it also represents the number of partial signature messages merged into a single message).
```

Comment:
didn't understand how partial messages relate to the graph

Reply:
Previously, I suggested combining the partial signature messages. The number of combined messages would be the same as the number of aggregated duties. Since you removed the part of the sip regarding psig msgs, no problem removing the text in the parenthesis.